packages/react/src/components/DetailsList/DetailsList.types.ts

```diff
@@ -433,6 +433,9 @@ export interface IColumn {
   /** If true, allow the column to be collapsed when rendered in justified layout. */
   isCollapsible?: boolean;
 
+  /** If true, column header will render a icon indicating column is sortable while unsorted */
+  isSortable?: boolean;
```

Comment:
I'm wondering if the fluent owners will have an issue with the naming of this prop. Technically, a column can be sortable even when isSortable is falsey. They may want it renamed to something like "showSortIconWhenUnsorted". @ThomasMichon will likely have the final say.

Reply:
Concur, and changed to the name you suggested. Open to reverting if the simpler name is preferred even with the lack of specifity